kext/FileNVRAM/FileNVRAM.cpp

```diff
@@ -217,7 +217,7 @@ void FileNVRAM::copyUnserialzedData(const char* prefix, OSDictionary* dict)
             
             if(prefix)
             {
-                size_t size = strlen(prefix) + sizeof(NVRAM_SEPERATOR) + strlen(key->getCStringNoCopy());
+                size_t size = strlen(prefix) + sizeof(NVRAM_SEPERATOR) + strlen(key->getCStringNoCopy()) +1;
```

Comment:
This is what happen:
`36C28AB5-6566-4C50-9EBD-CBB920F838§`
or
`D8F0CCF5-580E-4334-87B6-9FBBB831271D:NVRAFi`
Under /chosen/nvram all is ok, e.g is: `D8F0CCF5-580E-4334-87B6-9FBBB831271D:NVRAFile` and so on. 
Later I'll try to use strlen insead sizeoff as you suggested to see if is the same and let you know, although apparently we will have a char in less.
See you later